internal/migrate/create.go

```diff
@@ -5,21 +5,71 @@ package migrate
 
 import (
 	"database/sql"
+	"strconv"
 
 	"github.com/zeebo/errs"
 
 	"storj.io/storj/pkg/utils"
 )
 
+// Error is the default migrate errs class
+var Error = errs.Class("migrate")
+
 // DB is the minimal implementation that is needed by migration.
 type DB interface {
 	Begin() (*sql.Tx, error)
 	Schema() string
 	Rebind(string) string
 }
 
-// Error is the default migrate errs class
-var Error = errs.Class("migrate")
+type sqliteDB struct {
```

Comment:
I don't think this should be exposed. The DB `interface{}` is based off of dbx...

Reply:
yeah but we need somehow rebind sql params for postgres script

Reply:
It can be internal to the tests.

Reply:
ok, I assumed that at some point we will need postgres support. I will leave it internal.

Reply:
dbx.DB has those methods already defined